stores/utxo/src/main/java/com/bloxbean/cardano/yaci/store/utxo/storage/impl/UtxoStorageReaderImpl.java

```diff
@@ -73,8 +80,8 @@ public List<AddressUtxo> findUtxoByAddressAndAsset(String address, String unit,
                 .using(field(ADDRESS_UTXO.TX_HASH), field(ADDRESS_UTXO.OUTPUT_INDEX))
                 .where(ADDRESS_UTXO.OWNER_ADDR.eq(address))
                 .and(TX_INPUT.TX_HASH.isNull())
-                .and(field(ADDRESS_UTXO.AMOUNTS).cast(String.class).contains("\"unit\": \""+unit+"\""))
-                //.orderBy(order.equals(Order.desc) ? ADDRESS_UTXO.SLOT.desc() : ADDRESS_UTXO.SLOT.asc())  //TODO: Ordering
+                .and(isH2() ? field(ADDRESS_UTXO.AMOUNTS).cast(String.class).like("%\"unit\":\"" + unit + "\"%") : jsonGetAttribute(jsonGetElement(ADDRESS_UTXO.AMOUNTS, 0), "unit").cast(String.class).eq("\"" + unit + "\""))
```

Comment:
I am trying to get utxo for following address/unit on Preprod network, but it doesn't return anything.
I am using Postgres.

Address: addr_test1qz8gvl6n5wgk4zvu3ww4xfvthfc6vamrl28ructkcpsz0uy7d7vylw9x2vecxlt5qezanyecr9wrnyjx3teyaxn7hflqmzeau4
Unit: c6e65ba7878b2f8ea0ad39287d3e2fd256dc5c4160fc19bdf4c4d87e7447454e53

Why is index hard coded to "0" here?

```
jsonGetElement(ADDRESS_UTXO.AMOUNTS, 0)
```
Would it be able to match the unit if the corresponding utxo is at a non-zero index?